### PR TITLE
fix(webpack-dev-server): remove as it is not needed anymore

### PR DESCRIPTION
**Pull request by guidoiaquinti**
## Problem
Our version of `webpack-dev-server` is old.

## Changes
~This PR upgrade `webpack-dev-server` from version `3.11.2` to `4.9.2` (`yarn add webpack-dev-server`).~

This PR removes `webpack-dev-server` from our stack as we only use it with cypress and storybook, and those should manage their own launches.

## How did you test this code?
* PostHog works via `./bin/start` ✅ 
* `yarn storybook` ✅ 
* cypress: we will see in CI

### CODE REVIEW

**Comment by mariusandra** [comment]
I think the right answer is `yarn remove webpack-dev-server`. We only use webpack with cypress and storybook, and those should manage their own launches. You can also [remove the following lines](https://github.com/PostHog/posthog/pull/10120/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R23-R25) from `package.json`... or not, as I'll probably remove them soon enough with another PR.

File: package.json [hunk 1]
```diff
@@ -20,15 +20,6 @@
...
-        "start": "concurrently -n ESBUILD,TYPEGEN -c yellow,green \"yarn start-http\" \"yarn run typegen:watch\"",
-        "start-http": "yarn start-http:esbuild",
-        "start-http:esbuild": "yarn clean && yarn copy-scripts && node frontend/build.mjs --dev",
-        "start-http:webpack": "mkdir -p frontend/dist/ && cp -a frontend/public/* frontend/dist/ && npm run copy-scripts && webpack serve",
-        "start-https": "yarn start-https:webpack",
-        "start-https:webpack": "yarn start-http:webpack --https",
-        "start-docker": "yarn start-docker:esbuild",
-        "start-docker:esbuild": "yarn start-http:esbuild --host 0.0.0.0",
-        "start-docker:webpack": "yarn start-http:webpack --host 0.0.0.0",
         "start-ch-dev": "concurrently -n DOCKER,ESBUILD,TYPEGEN -c red,blue,green \"docker-compose -f docker-compose.dev.yml pull && CH_WEB_SCRIPT=./ee/bin/docker-ch-dev-backend docker-compose -f docker-compose.dev.yml up\" \"yarn run start-http --host 0.0.0.0\" \"yarn run typegen:watch\"",
```
Comment (mariusandra) [inline package.json:23]: Now that's a bit too much... We only want to yeet webpack. Fixing in a followup.

Comment (guidoiaquinti) [reply package.json:23]: I thought that too but if I'm not wrong I've removed exactly the lines you've highlighted in your previous comment

Comment (mariusandra) [reply package.json]: ah... the classic "Go buy bread. If they have eggs, buy six" --> comes home with 6 loafs of bread 🙈 

`yarn start` is still something we want

**Review by mariusandra — APPROVED** [review]

**Pull request merged**